aspnetcore/blazor/security/webassembly/azure-active-directory-groups-and-roles.md

```diff
@@ -414,12 +621,166 @@ public class BillingDataController : ControllerBase
 }
 ```
 
+::: moniker range=">= aspnetcore-5.0"
+
+### Packages
+
+Add package references to the *Server* app for the following packages:
+
+* [Microsoft.Graph](https://www.nuget.org/packages/Microsoft.Graph)
+* [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client)
+
+### Services
+
+In the *Server* app's `Startup.ConfigureServices` method, additional namespaces are required for the code in the `Startup` class of the *Server* app. The following set of `using` statements includes the required namespaces for the code that follows in this section and the default namespaces required by a server API app created from the hosted Blazor project template:
```

Comment:
Can we figure out a way to highlight what the additional and default namespaces are?

Reply:
Yes ... I'll do that on the next commit.